app/models/renewal/SendTheLargestAmountsOfMoney.scala

```diff
@@ -17,47 +17,92 @@
 package models.renewal
 
 import jto.validation.forms.UrlFormEncoded
-import jto.validation.{From, Rule, Write}
+import jto.validation.{From, Path, Rule, RuleLike, To, Write, WriteLike}
 import models.Country
-import play.api.libs.json.Json
-
-case class SendTheLargestAmountsOfMoney(
-                       country_1: Country,
-                       country_2: Option[Country] = None,
-                       country_3: Option[Country] = None
-                     ) {
-
-  def countryList = {
-    this.productIterator.collect {
-      case Some(Country(name, code)) => Country(name, code)
-      case x: Country => x
-    }
-  }
+import play.api.libs.json.{Json, Reads, Writes}
+import utils.TraversableValidators
 
-}
+case class SendTheLargestAmountsOfMoney (countries: Seq[Country])
 
-object SendTheLargestAmountsOfMoney {
+private sealed trait SendTheLargestAmountsOfMoney0 {
```

Comment:
Why this name?

Reply:
I just followed naming used in other places, but I could rename it to anything you want :)